resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Saw a yappy little dog today",
+            "Is sharpening {PRONOUN/m_c/poss} claws",
+            "Is hoping {PRONOUN/m_c/subject} find an opportunity to slip back to {PRONOUN/m_c/poss} nest",
+            "Can't wait to take a nap later",
+            "Had a nice chat with a cat from another clan at the border",
+            "Had a nice chat with a loner outside clan territory",
+            "Had a nice chat with a kittypet",
+            "Is on a walk out in the territory",
+            "Is having a nice conversation with the cats guarding camp",
+            "Got startled by a squirrel",
             "Saw a kittypet with the fluffiest fur imaginable today",
             "Saw a big scary dog today",
-            "Is looking forward to the next Gathering"
+            "Is drinking water from a nearby stream",
+            "Thought {PRONOUN/m_c/subject} saw a cat but turns out it was a twoleg object that looked like one"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+        ]
+    },
+	{
+        "id": "gen_to_kit",
+        "thoughts": [
+	"Is sitting near the camp entrance, keeping an eye out for sneaky kits",
+	"Is unable to be scared, no matter how much the kits try",
+        "Is keeping an eye on some rowdy kits"
+		
+	],
+	"main_status_constraint": [
+		"apprentice",
+        	"medicine cat apprentice",
+        	"mediator apprentice",
+        	"warrior",
+        	"medicine cat",
+        	"mediator",
+        	"deputy",
+        	"leader",
+        	"elder"
+	],
+	"random_status_constraint": [
+		"kit"
+		]
+	},
+	{
+        "id": "working_cat",
+        "thoughts": [
+            "Is guarding camp",
+            "Is sharpening {PRONOUN/m_c/poss} claws",
+            "Is exhausted after a patrol",
+            "Is wanting to go on a patrol",
+            "Is pretending to look busy...",
+            "Is helping out around camp in between {PRONOUN/m_c/poss} duties",
+            "Is wanting to train",
+            "Wants to go hunt",
+            "Doesn't want to go on dawn patrol",
+            "Doesn't want to go on dusk patrol",
+            "Is looking forward to the next Gathering",
+            "Missed a catch and is still sulking over it",
+            "Wishes {PRONOUN/m_c/subject} could take the day off",
+            "Is cursing a squirrel that just barely got away",
+            "Is wishing {PRONOUN/m_c/subject} had wings so {PRONOUN/m_c/subject} could catch birds easier"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "warrior",
+            "deputy",
+            "leader"
+        ],
+        "main_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "working_cat_warrior",
```

Comment:
i would suggest separating this into two though categories, one for random dep and one for random leader.  since some of these thoughts are deputy specific, you could end up in a situation where the clan has a leader but not a deputy, resulting in a cat having a thought about a deputy that doesn't exist.